### PR TITLE
book detail structure

### PR DESCRIPTION
**Pull request by jonnyalexbh**
## Summary
- book detail structure

## Trello Card
https://trello.com/c/Z0u6njth/28-detalle-de-libro-iii-estructura

## Screenshot
Captura de pantalla 2019-12-04 a la(s) 7 13 00 p  m

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -1,18 +1,42 @@
 .book-container {
-  border: 1px solid $light-gray;
+  background-color: $white;
+  box-shadow: 6px 6px 5px rgba($black, 0.4);
+  display: flex;
+  left: 50%;
+  margin: 20px;
```
Comment (betoobernal) [inline scss/detail.scss]: I believe that the margin is needless.

File: scss/detail.scss [hunk 2]
```diff
@@ -1,18 +1,42 @@
...
+  height: 340px;
+  width: 240px;
+}
+
+.book-information {
   display: flex;
   flex-direction: column;
-  height: 195px;
+  height: 220px;
   justify-content: space-between;
```
Comment (betoobernal) [inline scss/detail.scss]: it should not has `height` fixed. Try removing that `height` and you find other _value_ of `justify-content` property

File: scss/detail.scss [hunk 3]
```diff
@@ -1,18 +1,42 @@
...
   font-weight: bold;
   padding-bottom: 30px;
+  &::after {
+    background-color: $light-green;
+    content: '';
+    height: 4px;
+    left: 38%;
+    position: absolute;
+    top: 70px;
+    width: 450px;
```
Comment (betoobernal) [inline scss/detail.scss]: The pseudo-elements shouldn't have a width with absolute measure, you try with relative measures in relation to the element.

File: scss/detail.scss [hunk 4]
```diff
@@ -1,18 +1,42 @@
...
 
 .book-title {
   font-size: $text-big;
   font-weight: bold;
   padding-bottom: 30px;
+  &::after {
+    background-color: $light-green;
+    content: '';
+    height: 4px;
+    left: 38%;
```
Comment (betoobernal) [inline scss/detail.scss]: don't use relative measures for properties vertical and horizontal positions of a positioned pseudo-elements.

File: scss/detail.scss [hunk 5]
```diff
@@ -1,18 +1,42 @@
...
+    position: absolute;
```
Comment (betoobernal) [inline scss/detail.scss:35]: if a child node has `position: absolute;` its parent should have `position: relative;`.

This [article](https://css-tricks.com/absolute-relative-fixed-positioining-how-do-they-differ/) can help you

File: scss/detail.scss (hunk 1 above)
Comment (jonnyalexbh) [reply scss/detail.scss]: it is correct :)

File: scss/detail.scss (hunk 2 above)
Comment (jonnyalexbh) [reply scss/detail.scss]: ready

**Review by betoobernal — APPROVED** [review]

**Pull request merged**